apps/vc-api/test/vc-api/workflows/resident-card/resident-card-presentation.workflow.ts

```diff
@@ -11,7 +11,7 @@ import { VpRequestQueryType } from '../../../../src/vc-api/exchanges/types/vp-re
 export class ResidentCardPresentation {
   #workflowId = `b229a18f-db45-4b33-8d36-25d442467bab`;
   #callbackUrl: string;
-  queryType = VpRequestQueryType.presentationDefinition;
+  queryType = VpRequestQueryType.didAuth;
```

Comment:
This change is wrong, it should stay `presentationDefinition`. I debugged it and it is because line 142 in the test should be changed from using `issuanceWorkflow` to using `presentationWorkflow`.
```suggestion
  queryType = VpRequestQueryType.presentationDefinition;
```